lib/features/pfis/pfi_page.dart

```diff
@@ -14,139 +16,182 @@ class PfiPage extends HookConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final selectedPfi = useState<Pfi?>(null);
 
-    useEffect(
-      () {
-        Future.delayed(
-          Duration.zero,
-          () => ref.read(pfisProvider.notifier).reload(),
-        );
-        return null;
-      },
-      [],
-    );
+    useEffect(() {
+      Future.delayed(
+          Duration.zero, () => ref.read(pfisProvider.notifier).reload());
+      return null;
+    }, []);
```

Comment:
linting has been added in https://github.com/TBD54566975/didpay/commit/ae8f175a1672e8304d1564eedbc63b4e85af3422